compose/cli-command-compatibility.md

```diff
@@ -54,7 +54,7 @@ services:
     ports:
     - mode: ingress
       target: 80
```

Comment:
Is there a reason why `target` port is a number, but `published` port is a string?

Reply:
because you can use the range syntax to defined published port as we already mentioned in the [compose issue](https://github.com/docker/compose/issues/9306#issuecomment-1212185763)

Reply:
Ah ok, `target` cannot be a range, right? Thanks!